src/javascripts/ng-admin/Main/config/translate.js

```diff
@@ -39,6 +39,7 @@ export default function translate($translateProvider) {
       'DETAIL': 'Detail',
       'STATE_CHANGE_ERROR': 'State change error: {{ message }}',
       'STATE_FORBIDDEN_ERROR': 'A server 403 error occured: {{ message }}',
+      'GLOBAL_ERROR': 'Global error: {{ message }}',
```

Comment:
I would instead just specify `Error:`, as global is not really explicit for an end-user. :)